src/main/java/lotto/domain/LottoStore.java

```diff
@@ -0,0 +1,21 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class LottoStore {
+    private static final int LOTTO_PRICE = 1000;
```

Comment:
숫자를 사용할 때 습관적으로 언더바(_) 단위 분리를 해주시는게 숫자 단위 가독성에 도움이 되실거라 생각합니다 :)

```suggestion
    private static final int LOTTO_PRICE = 1_000;
```

---

Ranking enum 에 정의된 숫자들이 더 와닿겠네요 :) 같이 확인 해주세요~
`FIRST(6, 2_000_000_000)`

Reply:
보기 좋아졌네요 ㅎㅎ 눈이 편해졌습니다ㅎㅎ